packages/formatters-util/src/util/parseNumber.ts

```diff
@@ -1,18 +1,47 @@
+const replaceAllButLastOccurence = (input: string | string[], search: string) => {
+    // indexOf returns the first index, so flip the input to find the
+    // notator furthest back
+    const arrInput = Array.isArray(input) ? input.reverse() : input.split("").reverse();
+    const indexOfSearch = arrInput.indexOf(search);
+
+    return (
+        arrInput
+            // remove all instances of the notator except the last one
+            .filter((f, i) => {
+                if (f !== search) {
+                    return true;
+                }
+
+                return i === indexOfSearch;
+            })
+            // flip back and return as string
+            .reverse()
+            .join("")
+    );
+};
+
 export function parseNumber(input: string | number) {
     if (typeof input === "number") {
         return input;
     }
 
-    // fjern tegn som ikke kan tolkes som tall og erstatt kommadesimal
-    const strippedInput = input
-        .replace(/[^\d.,-]/g, "")
-        .replace(/[.,](\d{3})/g, "$1")
-        .replace(/,/g, ".");
-    const parsedNumber = Number.parseFloat(strippedInput);
+    // remove all spaces from number
+    const arrNumber = input.split("").filter((n) => n !== " ");
```

Comment:
En [String.prototype.replaceAll](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll) før `.split("")` er kanskje litt lettere å lese?

Reply:
`Property 'replaceAll' does not exist on type 'string'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2021' or later.ts(2550)`

Shall we?

Reply:
Let's!

Reply:
Da må vi legge inn ei litta .nvmrc eller noe som setter Node 16 som krav også. Jest får litt hetta av at .replaceAll() ikke er en funksjon i Node <16

Reply:
Vi har [.nvmrc](https://github.com/fremtind/jokul/blob/main/.nvmrc) med `16.14.0`

Reply:
Se det ja! Kanskje jeg burde sjekke før jeg sier noe neste gang 🙃